### PR TITLE
add php image documentation

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@adellape simple and non-controversial change.  please review/merge.

File: using_images/sti_images/php.adoc [hunk 1]
```diff
@@ -80,3 +80,9 @@ The following environment variables set their equivalent property value in the o
 * **OPCACHE_MEMORY_CONSUMPTION**
 ** The OPcache shared memory storage size
 ** Default: 16M
+
+You can also override the entire directory used to load the PHP configuration by setting:
+* **PHPRC**
```
Comment (adellape) [inline using_images/sti_images/php.adoc]: Needs a space here or it won't render as a bulleted list.

File: using_images/sti_images/php.adoc [hunk 2]
```diff
@@ -80,3 +80,9 @@ The following environment variables set their equivalent property value in the o
...
+  * Sets the path to the php.ini file
```
Comment (adellape) [inline using_images/sti_images/php.adoc]: If this is meant to be a sub-bullet, the line should start with `**` (see above lists).

File: using_images/sti_images/php.adoc [hunk 3]
```diff
@@ -80,3 +80,9 @@ The following environment variables set their equivalent property value in the o
...
+* **PHP_INI_SCAN_DIR**
+  * Path to scan for additional ini configuration files
```
Comment (adellape) [inline using_images/sti_images/php.adoc]: Same re: `**`.

**Pull request merged**